static/scss/cms/product.scss

```diff
@@ -25,6 +25,8 @@
 
       h1 {
         font-size: 3.5em;
+        text-transform: none;
+        letter-spacing: normal;
```

Comment:
Is this necessary after @ahmed-belal's PR #690 merges?

Reply:
**With letter-spacing: normal**
<img width="1080" alt="Screenshot 2020-06-19 at 00 14 55" src="https://user-images.githubusercontent.com/4043989/85062496-0393a180-b1c2-11ea-96ee-cc9242854e0d.png">

**Without letter-spacing**
<img width="1122" alt="Screenshot 2020-06-19 at 00 15 04" src="https://user-images.githubusercontent.com/4043989/85062515-0d1d0980-b1c2-11ea-9fc6-28436d8edd41.png">

@pdpinch If we want to go for the first screenshot then letter spacing would be required but it will be applied her section bootcamp title. If we go with the second one, then we can discard it.
Now let me know what you suggest?

Reply:
@umarmughal824 what @pdpinch is saying is that I'm working on a PR that removes all non-standard letter spacing from the CSS, so `letter-spacing: normal;` would not be required in your CSS anyway.

Reply:
yeah that would be required.

Reply:
@ahmed-belal actually I don't know the intention of `letter-space` removal so that why I had added both of the picture one with it and another one without it. So whatever @pdpinch choose we will finalize that one.